app/views/content_items/_body_with_related_links.html.erb

```diff
@@ -16,7 +16,7 @@
       <%= raw(@content_item.body) %>
       <% end %>
 
-      <% if @content_item.last_updated && @content_item.schema_name == "help_page" %>
```

Comment:
I think the && here implies this is _only_ shown for help_page documents, and can therefore be removed entirely, rather than cutting out the second test here?